src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>The sub section of Get Started page</description>
```

Comment:
The object descriptions need a little more detail. If I follow the structure correctly, this object is the child of the `GS_Admin_Checklist_Section__mdt` object. If so, maybe object description such as:

- GS_Admin_Checklist_Item__mdt : "NPSP Getting Started Page section items"
- GS_Admin_Checklist_Section__mdt: "NPSP Getting Started Page sections"